contact.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="contact.css">
+</head>
+
+<body background="IMG_1693.jpg">
+
+
+
+    <div class="Biography">
+        <p>My philosophy!</p>
+        <p>A photographer’s job is much more than taking just pictures, it's about finding the right tunes. I have an inherent ability to realize and capture compositions to show the world what the naked eye can sometimes not see. From doing fashion photography
+            to capturing the beautiful world that nature has given to man, I have done it all. </p>
+
+
+
+
+
+        <header>
+            <h2>Let's get in touch!</h2>
+
+        </header>
+    </div>
+    <div class="container">
+        <form action="/action_page.php">
+
+            <legend>
+                <h5>Contact Us</h5>
+            </legend>
+            <div class="row">
+                <div class="col-20">
+                    <label for="fname">First Name</label>
```

Comment:
Abbreviations are not very useful since they rely on everybody interpreting them the same.

Maybe someone doesn't see fname as firstname, there's no harm in writing it out except a couple of lines of extra code, but it's definitely more readable if it's written out.